variants/espino/pins_arduino.h

```diff
@@ -34,15 +34,16 @@
 static const uint8_t SDA = PIN_WIRE_SDA;
 static const uint8_t SCL = PIN_WIRE_SCL;
 
-static const uint8_t LED_BUILTIN = 2;
+#define LED_BUILTIN 2
 static const uint8_t LED_BUILTIN_R = 2;
 static const uint8_t LED_BUILTIN_G = 4;
 static const uint8_t LED_BUILTIN_B = 5;
 
-static const uint8_t BUILTIN_LED = 2;
+/*
 static const uint8_t BUILTIN_LEDR = 2;
 static const uint8_t BUILTIN_LEDG = 4;
 static const uint8_t BUILTIN_LEDB = 5;
 static const uint8_t BUILTIN_BUTTON = 0;
```

Comment:
maybe there should be a BUTTON_BUILTIN?

Reply:
Right. BUTTON_BUILTIN added, commented BUILTIN_* removed.